README.md

```diff
@@ -28,6 +28,10 @@ Shall the action fail if a license violation is detected? Default `false`
 
 **Required** Path to the config file for the license checker.
 
+### `generate-dash`
+
+Generate clearlydefined.input based on [ClearDefined Scheme](https://docs.clearlydefined.io/using-data)
```

Comment:
```suggestion
Generate clearlydefined.input based on [ClearDefined Schema](https://docs.clearlydefined.io/using-data)
```

The linked website does not mention anything about `clearlydefined.input`.  Also the parameter is called `generate-dash` how does this match the description?